sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableAsyncClient.java

```diff
@@ -111,7 +108,7 @@ public String getAccountName() {
     }
 
     /**
-     * returns Url of this service
+     * returns Url of this table
```

Comment:
Is it the URL of the table or is it the service?

Reply:
It's the URL to the table itself, determined by `serviceUri.resolve("/" + tableName)`